dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftRawLogsConnector.java

```diff
@@ -233,4 +250,29 @@ private void makeTasks(
       out.addTask(new JdbcSelectIntervalTask(file, query, interval));
     }
   }
+
+  /** Creates tasks to get Redshift cluster metrics from AWS CloudWatch API. */
+  private void makeClusterMetricsTasks(
+      ConnectorArguments arguments,
+      ZonedIntervalIterable intervals,
+      String filePrefix,
+      Class<? extends Enum<?>> headerEnum,
+      List<MetricConfig> metrics,
+      List<? super Task<?>> out) {
+    Optional<AWSCredentialsProvider> awsCredentials =
+        AbstractAwsApiTask.createCredentialsProvider(arguments);
+    if (awsCredentials.isEmpty()) {
+      return;
+    }
+
+    for (ZonedInterval interval : intervals) {
+      String file =
+          filePrefix
+              + DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(interval.getStartUTC())
```

Comment:
I would strongly suggest changing the format of the datetime to the one without colons, because they cause trouble during extraction from zip files in some environments.

Reply:
I changed Date format to `yyyy-MM-dd HH/mm/ss`.

Reply:
As discussed offline, this datetime format is about the timestamp that is inserted into the filename and it still needs changing.